src/mds/Server.h

```diff
@@ -214,6 +214,7 @@ class Server {
   void handle_client_setlayout(MDRequestRef& mdr);
   void handle_client_setdirlayout(MDRequestRef& mdr);
 
+  int64_t return_bytes(std::string& value_str);
```

Comment:
int: use std::string_view.

Reply:
For this I need to convert string to string_view at the time of function call i.e. a variable like `std::string_view val = value;` added. I hope that is fine?

Reply:
why, you can pass std::string to std::string_view.

Reply:
Okay so passing string_view by reference had some issues, reading upon I found it's anyways better to pass string_view by value and now it's working fine.